### ISSUE TITLE
github being rate limited again

### ISSUE THREAD

**Issue by romulasry** [fedora-infra/anitya#1846]
Who do we contact at github about this for release-monitoring.org?

**Comment by romulasry** [comment]
The primary rate limit for unauthenticated requests is 60 requests per hour.
[Primary rate limit for authenticated users](https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28#primary-rate-limit-for-authenticated-users)

You can use a personal access token to make API requests. Additionally, you can authorize a GitHub App or OAuth app, which can then make API requests on your behalf.

All of these requests count towards your personal rate limit of 5,000 requests per hour.

**Comment by Zlopez** [comment]
We have authenticated user, otherwise the ratelimiting would be much worse. I don't have any contact on GitHub to reach to.

**Comment by romulasry** [comment]
https://docs.github.com/en/support/contacting-github-support

**Comment by romulasry** [comment]
https://github.com/orgs/community/discussions/144808

**Comment by Zlopez** [comment]
Thank you for doing that :+1:

**Comment by lihaohong6** [comment]
Looks like there are no response from GitHub. A workaround would be to have 2 accounts and alternate between them to double the effective rate limit. GitHub may not like it, though, since this is intentionally evading their rate limit.

**Comment by damaestro** [comment]
Using a github app credentials is an option to increase quota: https://docs.github.com/en/apps/creating-github-apps/about-creating-github-apps/about-creating-github-apps

**Comment by damaestro** [comment]
> Using a github app credentials is an option to increase quota: https://docs.github.com/en/apps/creating-github-apps/about-creating-github-apps/about-creating-github-apps

This would require the release-monitoring app be installed into the target repos. So a non-starter.

**Comment by Justinzobel** [comment]
> I'm currently in contact with somebody from GitHub itself and trying to resolve the ratelimit with them. Hopefully that will help us at least increase the limit.

Any luck on this?

**Comment by Zlopez** [comment]
Got response for more data about Anitya and we will probably get a higher ratelimit. I'm just not sure how long it will take.

**Comment by wojnilowicz** [comment]
While trying to add a new project I get this error which effectively says that I can try again in 8 minutes. Is this known and expected?

**Comment by Zlopez** [comment]
Yes, unfortunately I didn't get any exception for Anitya with GitHub. I'm working on it, but right now we are rate limited as every other service using API.
